growth/too/flask.py

```diff
@@ -52,6 +51,14 @@
     for dropin_file in dropin_files:
         app.config.from_pyfile(os.path.join('application.cfg.d', dropin_file))
 
+if "GATTINI_USER" in app.config:
```

Comment:
It won't even try to connect to the database until you try to access the engine, so you need not check for the configuration variable to be present. Just provide a reasonable default value for the SQL bind URL, like we did in the past for the GROWTH marshal.

Reply:
Take a look at this file just prior to commit ab3a385fbd98a2b8985c9bf6f846a3cabb414a35.